static/js/template/services/template.service.js

```diff
@@ -354,88 +355,91 @@
+                            '<audio class="audio-container" ng-src="{{item.options}}" audioplayer controls style="margin-bottom:8px;">' +
                             '<p>Your browser does not support the <code>audio</code> element.</p> </audio>';
                         return html;
                     },
                     toEditor: function () {
                         var html = '<h1 class="md-subhead" ng-bind="item.label"></h1>' +
-                            '<audio class="audio-container" ng-src="{{item.options}}" audioplayer controls style="margin-bottom:8px;">'+
-                            '<p>Your browser does not support the <code>audio</code> element.</p> </audio>'+
-                            '<div class="_item-properties">'+itemToolbar+'<md-input-container>' +
+                            '<audio class="audio-container" ng-src="{{item.options}}" audioplayer controls style="margin-bottom:8px;">' +
+                            '<p>Your browser does not support the <code>audio</code> element.</p> </audio>' +
+                            '<div class="_item-properties">' + itemToolbar +
+                            '<md-input-container>' +
                             '<label>Heading</label>' +
                             '<input ng-model="item.label">' +
                             '</md-input-container>' +
                             '<md-input-container>' +
                             '<label>Audio URL</label>' +
                             '<input ng-model="item.values" ng-required>' +
-                            '</md-input-container></div>';
+                            '</md-input-container>' +
+                            '</div>';
                         return html;
                     }
                 },
                 /*{
-                    tooltip: "Video Container",
-                    layout: 'column',
-                    data_source: null,
-                    role: 'display',
-                    label: '',
-                    name: "Video Container",
-                    icon: 'play_circle_outline',
-                    type: 'video'
-                },
+                 tooltip: "Video Container",
+                 layout: 'column',
+                 data_source: null,
+                 role: 'display',
+                 label: '',
+                 name: "Video Container",
+                 icon: 'play_circle_outline',
+                 type: 'video'
+                 },
+                 {
+                 tooltip: "External Content (iFrame)",
+                 layout: 'column',
+                 data_source: null,
+                 role: 'display',
+                 label: '',
+                 name: "iFrame",
+                 icon: 'web',
+                 type: 'video'
+                 },*/
                 {
-                    tooltip: "External Content (iFrame)",
+                    name: "Remote Content",
```

Comment:
use the new way of writing the templates, I moved them to files because it's getting harder to maintain as these are growing, see media-edit and media templates it's very easy to plug iFrame in all the rest will work out of the box